composer.json

```diff
@@ -37,27 +36,28 @@
             "homepage": "https://github.com/jaylinski"
         }
     ],
+    "homepage": "https://github.com/friends-of-phpspec/phpspec-code-coverage",
+    "support": {
+        "issues": "https://github.com/friends-of-phpspec/phpspec-code-coverage/issues",
+        "source": "https://github.com/friends-of-phpspec/phpspec-code-coverage",
+        "docs": "https://github.com/friends-of-phpspec/phpspec-code-coverage#phpspec-code-coverage"
+    },
     "require": {
         "php": ">= 7.3",
         "phpspec/phpspec": "^6.0 || ^7.0",
         "phpunit/php-code-coverage": "^9.0"
     },
+    "require-dev": {
+        "drupol/php-conventions": "^5.0"
```

Comment:
@drupol Can you help me with this? Your latest grumphp config does all sorts of new things... I had to update it in order to support PHP 8.1, but v5 doesn't support 7.3 anymore.

Also: I'm not sure if the whole license-stuff should be rewritten. Can this be disabled in our grumphp config?

Reply:
@drupol No problem. Do you want to remain as "friends-of-phpspec"-org-member?

Also: do you still maintain your repository https://github.com/friends-of-phpspec/phpspec-requires?

Reply:
> @drupol No problem. Do you want to remain a "friends-of-phpspec"-org-member?

I already asked @shulard to remove me from the organisation a couple of weeks ago.

> Also: do you still maintain your repository https://github.com/friends-of-phpspec/phpspec-requires?

No, I don't maintain anything related to PHPSpec. Sorry. It was a hard decision to take, but it is what it is... and I'm not planning to use PHPSpec anymore either.

Reply:
Ok I've just archived the phpspec-requires repository. @drupol maybe you can mark the package deprecated on packagist ?

Reply:
Done !